### DISCUSSION TITLE
Railway tagging requested

### DISCUSSION THREAD

**Discussion by codemaster2b** [BattlesnakeOfficial/feedback#236] [Feature Requests]
Hello,

I host my snakes using Railway (https://railway.app/) but I cannot tag my snake using this term.

Thanks,
Jon Butterfield (jdb1662)

**Comment by chris-bsnake** [comment]
Hi Jon,

Thanks for the request, I have manually added it to the system. We don't have an icon configured in the system for it yet but you can tag your Battlesnake with Railway as a platform provider

**Comment by bvanvugt** [comment]
Added an icon! 👍
